### PR TITLE
Added docker-compose configure to run estuary stack on local developm…

### LINKED ISSUE #2: introduce - docker-compose
# Enhancement Request
Now that we have containerize the nodes, we need to create a docker-compose to spin main estuary with shuttles with a single docker-compose command.

There is a preliminary docker-compose yaml file to do this but there are still a few things to do to complete:

1. grab the auth key from the main estuary
2. assign the generated token to the shuttles

# Possible approach
1 - we can grab the tokens and parse them using regex. We can then assign this as an environment variable for each shuttle.

### PR DESCRIPTION
**Pull request by 10d9e**
…ent machine

### CODE REVIEW

**Comment by 10d9e** [comment]
close #2

**Review by alvin-reyes — APPROVED** [review]
LGTM! This is awesome!

We should definitely put more time on extending this. Setup a K8s cluster and isolate this repo as infra code so we can introduce a standard deployment / provisioning for our customers and possibly some one-click deploy that we can hook to bitnami, digital ocean droplets, AWS/EKS or Azure. 

Lots to be done but what we have here is a great start. Thank you!

File: docker-compose.yml [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
-      - 3004:3004
-    env_file:
-      - ./.env 
+      - "3004:3004"
+      - "3004:3004/udp"
+  
   estuary-shuttle:
     build:
       context: ./estuary-shuttle
       dockerfile: Dockerfile
```
Comment (gmelodie) [inline docker-compose.yml:22]: Do we want to expose 3005 (default port for the shuttle)?

**Review by gmelodie — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (gmelodie) [inline docker-compose.yml:22]: Do we want to expose port 3005 for the shuttle?

Comment (10d9e) [reply docker-compose.yml:22]: Good catch

**Pull request merged**